### PR TITLE
Step7 Create CRUD

### PR DESCRIPTION
**Pull request by TaitoItou**
## 概要
Based on the task model created in Step 6, let's make it possible to register, update, and delete tasks.

## 実装内容
- [ ] create a controller and a view by using `rails generate` command
- [ ] display a flash message on the screen after each creation, update, and deletion
- [ ] edit `routes.rb` so that `http://localhost:3000/` will show the task list screen
- [ ]  install  Rubocop or fablicop by using Gemfile

## memo
There are some Step6 contents. 
~I will merge taito_ito_step6 branch when the branch is ready to merge.~
↑ already merged

### CODE REVIEW

File: myapp/app/views/layouts/_tasks_flash.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<% if flash[:success].present? %>
+  <div class="alert alert-success" role="alert"><%= flash[:success] %></div>
+<% end %>
+<% if flash[:danger].present? %>
+  <div class="alert alert-danger" role="alert"><%= flash[:danger] %></div>
+<% end %>
```
Comment (mt-ytaher) [inline myapp/app/views/layouts/_tasks_flash.html.erb]: Add a new line at the end of the file. Its Rakuma's coding practice

File: myapp/.rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+inherit_gem:
+  fablicop:
+    - "config/.base_rubocop.yml"
```
Comment (mt-ytaher) [inline myapp/.rubocop.yml:3]: nice 👍

File: myapp/app/views/tasks/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+<% @tasks.each do |task| %>
+    <tr>
+        <td><%= task.id %></td>
+        <td><%= link_to task.title, task%></td>
+        <td><%= link_to 'Edit', edit_task_path(id: task.id)%></td>
+        <td><%= link_to 'Delete', task, method: :delete, data: { confirm: 'Are you sure about it?' } %></td>
+    </tr>
+<% end %>
+</table>
+<p><%= link_to 'create', new_task_path %></p>
```
Comment (mt-ytaher) [inline myapp/app/views/tasks/index.html.erb]: same as above "new line at the end of the file"

File: myapp/app/views/tasks/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      <%= form.text_field :description %>
+    </div>
+
+    <div class="actions">
+      <%= form.submit :submit %>
+    </div>
+  <% end %>
+</div>
+
+<%= link_to 'Back', tasks_path %>
```
Comment (mt-ytaher) [inline myapp/app/views/tasks/new.html.erb]: same as above "new line at the end of the file"

**Review by mt-ytaher — CHANGES_REQUESTED** [review]

File: myapp/app/views/layouts/_tasks_flash.html.erb (hunk 1 above)
Comment (TaitoItou) [reply myapp/app/views/layouts/_tasks_flash.html.erb]: Thank you.
I'll make sure to add a new line at the end of the file!

**Review by mt-ytaher — APPROVED** [review]

**Comment by mt-ytaher** [comment]
LGTM

lgtm

**Pull request merged**